execute/executetest/source.go

```diff
@@ -51,7 +51,7 @@ func (src *FromProcedureSpec) AddTransformation(t execute.Transformation) {
 }
 
 func (src *FromProcedureSpec) Run(ctx context.Context) {
-	id := execute.DatasetID(uuid.NewV4())
+	id := execute.DatasetID(uuid.Must(uuid.NewV4()))
```

Comment:
I don't know much about how UUID's are generated, but if the docs panic when there's an error, that seems like a good way to do this.

Reply:
@lyondhill can you add a todo here about answering the question of when this panics? I think @fchikwekwe brings up a good point here and, while I don't want to delay the PR, it is a question I would like an answer to.

Reply:
Will do.